exonum-light-client/src/test/java/com/exonum/client/Blocks.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.client;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.client.response.Block;
+import java.time.ZonedDateTime;
+
+final class Blocks {
+  static final String BLOCK_1_JSON = "{\n"
+      + "        'proposer_id': 3,\n"
+      + "        'height': 1,\n"
```

Comment:
As they usually go as 1, 2, 3, I'd set heights accordingly (now they are 1, 21, 16, but must be in desc order, mustn't they?)